CharlieBackend.Business/Services/MentorService.cs

```diff
@@ -233,7 +233,27 @@ public async Task<Result<MentorDto>> DisableMentorAsync(long mentorId)
 
             if (!await _accountService.DisableAccountAsync(accountId.Value))
             {
-                return Result<MentorDto>.GetError(ErrorCode.NotFound, "This account is already disabled.");
```

Comment:
due to changing error code update controllers response CODE

Reply:
Solved for each controller in my pr. https://github.com/ita-social-projects/WhatBackend/pull/420/commits/1c8f0993b0fe87708ba1c1e33357da216b71f40e